`curate-maintain` Add content for the Content section (#35)
open-education-hub/methodology: chapters/curate-maintain/content/reading/README.md

### Content

The content maintainer is responsible for overseeing different aspects of the open educational resource, like content management, accepting and reviewing contributions, or community building.

It is important for such a resource to clearly identify who the maintainer is, for the purpose of transparency and good communication.

The content is typically organized in a `Git` repository, which has a series of advantages like version control, branching, history, etc.

> It is also recommended to use a managed development platform like `GitHub` or `GitLab`, which is built on top of `Git`, while also providing additional useful features, like code review, issues, pull requests, integrations, and others.

Reviewer: ```suggestion
It is also recommended using a managed development platform like `GitHub` or `GitLab`, which is built on top of `Git`, while also providing additional useful features, like code review, issues, pull requests, integrations, and others.
```

The content should be published under a license.

> An example of a widely used license for public education resources is `Creative Commons`.

Reviewer: I would rephrase this to state that we strongly encourage (and use) CC licencing

#### Contributions

The content maintainer should foster a welcoming environment for outside contributions.

> This can be achieved by:

Reviewer: ```suggestion
This can be achieved by:

```

> - establishing clear contribution guidelines (for example in a `CONTRIBUTING.md` file)

Reviewer: ```suggestion
- Establishing clear contribution guidelines (for example in a `CONTRIBUTING.md` file)
```

> - labeling issues with tags such as `help wanted` or `good first issue`, to help newcomers find tasks to contribute

Reviewer: ```suggestion
- Labeling issues with tags such as `help wanted` or `good first issue`, to help newcomers find tasks to contribute
```

> - acknowledging the work of contributors (for example in the repository `README.md` file or in any other suitable place)

Reviewer: ```suggestion
- Acknowledging the work of contributors (for example in the repository `README.md` file or in any other suitable place)
```

There should also be an established process for handling contributions whenever they arrive.

> Some steps that the maintainer should take are:

Reviewer: ```suggestion
Some steps that the maintainer should take are:

```

> - review the contribution as soon as possible.

Reviewer: ```suggestion
- Review the contribution as soon as possible.
```

Depending on the team size, this can be done by the maintainer, or it could be delegated to other members of the content development team.

> Tools like `GitHub` have features that can help with this: an issue can have a person designated as assignee, while a pull request can have one or multiple reviewers.

Reviewer: ```suggestion
Tools like `GitHub` have features that can help with this: an issue can have a person designated as assignee, while a pull request can have one or multiple reviewers and assignees.
```

> - engage in a constructive discussion with the contributor to clarify any questions or concerns.

Reviewer: ```suggestion
- Engage in a constructive discussion with the contributor to clarify any questions or concerns.
```

Provide feedback and guidance for improvement if the contribution needs changes.

> - ensure that the contribution follows the style guidelines.

Reviewer: ```suggestion
- Ensure that the contribution follows the guidelines.
```
There can be many guidelines, not necessarily style-related.

This can be partially automated using linters and automated checkers, together with features like `GitHub Actions`.

> - if applicable, test the contribution to ensure it works as intended.

Reviewer: ```suggestion
- If applicable, test the contribution to ensure it works as intended.
```

> - thank the contributor for their effort.

Reviewer: ```suggestion
- Thank the contributor for their effort.
```

#### Creating a repository

> If a new content repository needs to be created from scratch, a good starting point is to clone the [oer-template](https://github.com/open-education-hub/oer-template) repository.

Reviewer: I would add a short phrase on what the `oer-template` contains. Just say that it has a basic repository structure with all the tools included, and any kind of content can be added on top of that.

> If a new content repository needs to be created from scratch, a good starting point is to clone the [oer-template](https://github.com/open-education-hub/oer-template) repository.

Reviewer: ```suggestion
If a new content repository needs to be created from scratch, a good starting point is to clone the [`oer-template`](https://github.com/open-education-hub/oer-template) repository.
```

### Content

The content maintainer is responsible for overseeing different aspects of the open educational resource, like content management, accepting and reviewing contributions, or community building.

> It is important for such a resource to clearly identify who the maintainer is, for the purpose of transparency and good communication.

Reviewer: What exactly does `such a resource` refer to?

Author: An open education resource.

Reviewer: Got it, it's fine.

> The content is typically organized in a `Git` repository, which has a series of advantages like version control, branching, history, etc.

Reviewer 2: ```suggestion
The content is typically organized in a versioned system control repository such as `Git`, which has a series of advantages like version control, branching, history, etc.
```

It is also recommended using a managed development platform like `GitHub` or `GitLab`, which is built on top of `Git`, while also providing additional useful features, like code review, issues, pull requests, integrations, and others.

The content should be published under a license.

> At `Open Education Hub` we employ the `Creative Commons` license ourselves and strongly recommend its usage.

Reviewer: ```suggestion
At `Open Education Hub` we employ the [`Creative Commons`](https://en.wikipedia.org/wiki/Creative_Commons_license) license ourselves and strongly recommend its usage.
```

#### Contributions

The content maintainer should foster a welcoming environment for outside contributions.
This can be achieved by:

> - Establishing clear contribution guidelines (for example in a `CONTRIBUTING.md` file)
> - Labeling issues with tags such as `help wanted` or `good first issue`, to help newcomers find tasks to contribute
> - Acknowledging the work of contributors (for example in the repository `README.md` file or in any other suitable place)

Reviewer: ```suggestion
- Establishing clear contribution guidelines (for example in a `CONTRIBUTING.md` file).
- Labeling issues with tags such as `help wanted` or `good first issue`, to help newcomers find tasks to contribute.
- Acknowledging the work of contributors (for example in the repository `README.md` file or in any other suitable place).
```
Add dots at the end of sentences.

There should also be an established process for handling contributions whenever they arrive.
Some steps that the maintainer should take are:

> - Review the contribution as soon as possible.
> Depending on the team size, this can be done by the maintainer, or it could be delegated to other members of the content development team.
> Tools like `GitHub` have features that can help with this: an issue can have a person designated as assignee, while a pull request can have one or multiple reviewers and assignees.

Reviewer 3: ```suggestion
- Review the contribution as soon as possible.

  Depending on the team size, this can be done by the maintainer, or it could be delegated to other members of the content development team.
  Tools like `GitHub` have features that can help with this: an issue can have a person designated as assignee, while a pull request can have one or multiple reviewers and assignees.
```

- Engage in a constructive discussion with the contributor to clarify any questions or concerns.
Provide feedback and guidance for improvement if the contribution needs changes.
- Ensure that the contribution follows the guidelines.
This can be partially automated using linters and automated checkers, together with features like `GitHub Actions`.
- If applicable, test the contribution to ensure it works as intended.

> - Thank the contributor for their effort.

Reviewer: Maybe add that the maintainer should make sure contributions are merged, and do not end up lingering for too long?

> - Thank the contributor for their effort.

Reviewer 3: Buy him something nice once in a while.
